client/lib/url/index.js

```diff
@@ -20,4 +21,4 @@ function isExternal( url ) {
 	return isOutsideCalypso( url ) && ! startsWith( url, '//wordpress.com' );
 }
 
-export default { isOutsideCalypso, isExternal };
+export default { isOutsideCalypso, isExternal, support };
```

Comment:
If you're export `support` like that here, I think one would expect you to import it via `{ support } = require( 'lib/url' )` (you don't _have_ to, of course).
If you prefer to do it more directly, I'm totally cool with that, but I don't see the point of exporting the links here then.
My preferred solution is to keep the exports here to the helper methods and import the urls directly from their modules.